src/main/java/io/ebeaninternal/server/deploy/parse/AnnotationFields.java

```diff
@@ -282,12 +297,8 @@ private void readField(DeployBeanProperty prop) {
       generatedPropFactory.setUpdateTimestamp(prop);
     }
 
-    if (get(prop, WhoCreated.class) != null) {
-      generatedPropFactory.setWhoCreated(prop);
-    }
-    if (get(prop, WhoModified.class) != null) {
-      generatedPropFactory.setWhoModified(prop);
-    }
+    initWhoCreated(prop);
+    initWhoModified(prop);
 
```

Comment:
True, although these methods are only called during initialisation of EbeanServer (bootup time) and not what I'd call runtime (for every insert or update times every BindableAssocOne).  So you might be right but at this stage I haven't looked at performance optimisation of "bootup time" much and I'm happy with initWhoCreated() as it is called from 2 places etc.  

So for me I think this is all good.